### PR TITLE
suthesh/remove-careers-sitemap

### PR DESCRIPTION
**Pull request by suthesan-deriv**
Changes:

-   removed localized career page links from sitemap

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by sean-binary — DISMISSED** [review]

File: gatsby-config.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ module.exports = {
                 ],
                 serialize: ({ site, allSitePage }) =>
                     allSitePage.edges.map((edge) => {
+                        const ignore_localized_regex = /careers/;
```
Comment (sean-binary) [inline gatsby-config.js:58]:
```suggestion
                        const careers_regex = /careers/;
```

Comment (sean-binary) [reply gatsby-config.js:58]: Maybe we can call this by its name perhaps? And then using it in `ignore_localized`

File: gatsby-config.js [hunk 2]
```diff
@@ -82,14 +83,15 @@ module.exports = {
 
                         languages.push('x-default')
                         languages.splice(languages.indexOf('ach'), 1)
-                        const links = languages.map((locale) => {
+                        const ignore_localized = current_page.match(ignore_localized_regex);
+                        let links;
+
+                        links = languages.map((locale) => {
```
Comment (sean-binary) [inline gatsby-config.js:89]:
```suggestion
                        const links = languages.map((locale) => {
```

Comment (sean-binary) [reply gatsby-config.js:89]: Can we declare it here instead?

File: gatsby-config.js (hunk 1 above)
Comment (suthesan-deriv) [reply gatsby-config.js:58]: Was planning on using this to probably add more pages into this list in future

File: gatsby-config.js (hunk 2 above)
Comment (suthesan-deriv) [reply gatsby-config.js:89]: Yup we totally should 😂

**Pull request merged**